README.md

```diff
@@ -232,6 +233,4 @@ I would like to thank Microverse and the its global community.
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional

Reply:
Thank you for your feedback